### ISSUE TITLE
InlineRuntime no longer works

### ISSUE THREAD

**Issue by maxtaco** [maxtaco/coffee-script#173]
``` code
> iced -I inline x.iced 
ReferenceError: InlineRuntime is not defined
  at IcedRuntime.<anonymous> (/Users/max/opt/lib/node_modules/iced-coffee-script/lib/coffee-script/nodes.js:3197:20)
  at IcedRuntime.compileNode (/Users/max/opt/lib/node_modules/iced-coffee-script/lib/coffee-script/nodes.js:3219:10)
  at Block.exports.Block.Block.compileNode (/Users/max/opt/lib/node_modules/iced-coffee-script/lib/coffee-script/nodes.js:437:35)
  at Block.exports.Block.Block.compileWithDeclarations (/Users/max/opt/lib/node_modules/iced-coffee-script/lib/coffee-script/nodes.js:533:19)
  at Block.exports.Block.Block.compileRoot (/Users/max/opt/lib/node_modules/iced-coffee-script/lib/coffee-script/nodes.js:505:24)
  at Block.exports.Block.Block.compileToFragments (/Users/max/opt/lib/node_modules/iced-coffee-script/lib/coffee-script/nodes.js:422:21)
  at /Users/max/opt/lib/node_modules/iced-coffee-script/lib/coffee-script/coffee-script.js:68:62
  at /Users/max/opt/lib/node_modules/iced-coffee-script/lib/coffee-script/coffee-script.js:38:19
```

**Comment by zapu** [comment]
Hopefully fixed in https://github.com/maxtaco/coffee-script/pull/176

**Comment by zapu** [comment]
Do we close this one or is it sill not perfect? It would be cool to try to update the iced website and see if we can get the compile-in-browser working.

**Comment by maxtaco** [comment]
Closed in #176
